test/extended/security/scc.go

```diff
@@ -271,7 +271,10 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
 
 	g.It("TestPodDefaultCapabilities", func() {
 		g.By("Running a restricted pod and getting it's inherited capabilities")
-		pod, err := exutil.NewPodExecutor(oc, "restrictedcapsh", image.ShellImage())
+		// This test should use image.ShellImage but this requires having a local image
+		// registry, which not all deployment types have. Using LDAP test image guarantees
+		// capsh presence.
+		pod, err := exutil.NewPodExecutor(oc, "restrictedcapsh", image.OpenLDAPTestImage())
```

Comment:
@stlaz for taking a look

Reply:
@openshift/openshift-team-auth PTAL

Reply:
@pacevedom is there a less complex image than LDAP? Something on the same level of complexity as the shell image? Complexity = minimum rpm installation/configuration inside the image. I assume you choosed OpenLDAPTestImage since it's publicly available from `quay.io/openshifttest/ldap:1.2`. I wonder whether we run these tests in a disconnected environment.